packages/visual-dev/src/components/Alert/Alert.tsx

```diff
@@ -22,7 +22,13 @@ export interface StyleProps {
   /**
    * Alert type that affects border, background, and text colours
    */
-  type: "critical" | "warning" | "success" | "info";
+  type:
```

Comment:
Another option here would be possibly to have a `text` boolean prop? It would allow all banner types to have a text mode

Reply:
Going to leave this as a nice to have